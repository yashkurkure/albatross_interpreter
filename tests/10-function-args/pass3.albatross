fun f1 int (argument int) {
  return 0;
}

fun f2 void() {
  var v int := 473;

  f1(v);
}

return 0;
