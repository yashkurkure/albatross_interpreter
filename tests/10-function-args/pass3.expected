Function declared "f1" returns int
	Argument "argument" type int position 0
Function declared "f2" returns void
	Local variable "v" type int position 0
Function called "f1" returns int
